runtime/indent/scala.vim

```diff
@@ -501,7 +501,7 @@ function! GetScalaIndent()
           let ind = indent(completeLine)
         endif
       else
-        " This is the only part that's different from from the '{', '}' one below
+        " This is the only part that's different from the '{', '}' one below
```

Comment:
We don't maintain these files (and hence don't make any changes directly). Please contact the maintainer listed at the top of the file.